penguin/app/components/enjoy.js

```diff
@@ -0,0 +1,36 @@
+var React = require('react-native');
+var styles = require('./Styles');
+var IP_address = require('../../environment.js').IP_address;
+
+
+var {
+  Component,
+  View,
+  Text,
+  Image
+} = React;
+
+class Enjoy extends Component{
+  constructor(props) {
+    super(props);
+  }
+
+  render(){
+    var imageLinkMe = `${IP_address}/users/${this.props.username}/profilePhoto`;
+    var imageLinkMatch = `${IP_address}/users/${this.props.match.username}/profilePhoto`;
+    return (
+      <View style={styles.mainContainer}>
+        <Image source={{uri:"http://blogs-images.forbes.com/briansolomon/files/2014/05/cmg_fd_steakbowl_300-e1400181721565.jpg"}}
```

Comment:
I like the image background. The lag time was pretty slow on my end bc i have slow internet. 

Can you save a local copy to the Xcode project?

This'll walk you through it--- [Adding files to Xcode](https://developer.apple.com/library/ios/recipes/xcode_help-structure_navigator/articles/Adding_an_Existing_File_or_Folder.html)